docs/contributing/managing-pull-requests.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Managing Pull Requests
+---
+
+## Introduction
+
+If you'd like to help out managing PRs on the gatsby repo on GitHub, this document is for you. We'll go over conventions we prefer, what we like to check for on various types of pull requests, permissions and guidelines on how to leave feedback.
```

Comment:
One thing I just noticed is that this doc deviates from the recommended "you" and "you're" style that we recommend in docs to focus on the reader. But in this instance, managing PRs is very much reliant on the core team's guidance and permissions, so I think it's fine. But something to keep in mind for future docs!